amundsen_application/static/js/components/SearchPage/SearchFilter/styles.scss

```diff
@@ -7,19 +7,19 @@
 
   .search-filter-section-header {
```

Comment:
Why would we un-nest styles? Once upon a time we intentionally moved towards nesting styles to decrease collisions given that we don't have anything else set up right now to assist with that, and have ended up with a few bugs because of that.

Reply:
Yeah, some nesting is OK, and I personally keep it at two levels max.
The Sass guidelines suggest to limit it to just one.

The problem with nesting is that generates styles with really high specificity, so when you need to override them, you will be forced to use "!important" (as I just had to do here).

If you want, I can help on moving us into BEM(https://9elements.com/bem-cheat-sheet/), it is relatively easy and eliminates all collisions at once.

Reply:
I see. I don't mind this level of nesting. Though if the issue is that it creates problems when overriding then that makes a case for how we approach the styles for items in `common` that are meant to be reuseable, and wouldn't particularly invest the time into these highly specific components. 

In regards to the posted link, I won't be able to form an opinion until we do some kind of semi-formal evaluation of the applications needs and weigh them against other approaches regarding css collisions.